Week1/database.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+let mysql = require('mysql');
+let connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password: 'hyfpassword',
+    database: 'userdb'
+});
+
+let database = 'create database IF NOT EXISTS new_world';
+let useDatabase = 'use new_world';
+
+let tableCountry = ` CREATE TABLE IF NOT EXISTS country(
+    name CHAR(20),
+    population INT,
+    ends_land CHAR(20),
```

Comment:
There is no country_code field here.
Please check all queries carefully before you design the table.